src/plugins/regmon/regmon.h

```diff
@@ -111,9 +111,9 @@
 class regmon: public plugin
 {
 public:
-    drakvuf_trap_t traps[3] =
+    drakvuf_trap_t traps[32] =
```

Comment:
I don't think we need to allocate 32 traps here, it looks like you are using only 13 of them.

Reply:
Done.